src/basic/virt.c

```diff
@@ -235,8 +235,28 @@ static int detect_vm_dmi(void) {
 
         /* The DMI vendor tables in /sys/class/dmi/id don't help us distinguish between Amazon EC2
          * virtual machines and bare-metal instances, so we need to look at SMBIOS. */
-        if (r == VIRTUALIZATION_AMAZON && detect_vm_smbios() == SMBIOS_VM_BIT_UNSET)
-                return VIRTUALIZATION_NONE;
+        if (r == VIRTUALIZATION_AMAZON) {
```

Comment:
Unnecessary `{`

Reply:
I disagree, I find nested statements like this without { to be error prone and generally sore to the eye

Reply:
OK. In this case, I personally does not request to drop it strongly. But note that our coding style says we should drop `{` in such case 'in general'.

Reply:
I've pushed an update without this one, if you do feel that strongly about it I'll fix it up too